scripts/core/CONTRIB.rst

```diff
@@ -219,13 +219,38 @@ implement your experimental feature.
 Naming Convention
 -----------------
 
+The following naming conventions must be followed:
+
+## --validate=off
+    *   All functions must be prefixed with ``${x}``
+    *   All functions must use camel case ``${x}ObjectAction`` convention
+    *   All macros must use all caps ``${X}_NAME`` convention
+    *   All structures, enumerations and other types must follow ``${x}_name_t`` 
+        snake case convention
+    *   All structure members and function parameters must use camel case 
+        convention
+    *   All enumerator values must use all caps ``${X}_ENUM_ETOR_NAME`` 
+        convention
+    *   All handle types must end with ``handle_t``
+    *   All descriptor structures must end with ``desc_t``
+    *   All property structures must end with ``properties_t``
+    *   All flag enumerations must end with ``flags_t``
+## --validate=on
+
+The following coding conventions must be followed:
+
+    *   All descriptor structures must be derived from ${x}_base_desc_t
+    *   All property structures must be derived from ${x}_base_properties_t
+    *   All function input parameters must precede output parameters
+    *   All functions must return ${x}_result_t
```

Comment:
I'd like to avoid making this a block quote

```suggestion
## --validate=off
*   All functions must be prefixed with ``${x}``
*   All functions must use camel case ``${x}ObjectAction`` convention
*   All macros must use all caps ``${X}_NAME`` convention
*   All structures, enumerations and other types must follow ``${x}_name_t`` 
    snake case convention
*   All structure members and function parameters must use camel case 
    convention
*   All enumerator values must use all caps ``${X}_ENUM_ETOR_NAME`` 
    convention
*   All handle types must end with ``handle_t``
*   All descriptor structures must end with ``desc_t``
*   All property structures must end with ``properties_t``
*   All flag enumerations must end with ``flags_t``
## --validate=on

The following coding conventions must be followed:

*   All descriptor structures must be derived from ${x}_base_desc_t
*   All property structures must be derived from ${x}_base_properties_t
*   All function input parameters must precede output parameters
*   All functions must return ${x}_result_t
```

Reply:
Fixed.